### PR TITLE
Remove healthmodelpanel source from project

### PR DESCRIPTION
**Pull request by nielsams**
Treating the healthmodelpanel as a third party component installed through NPM. This will make it easier to manage across different repos and cause less battles with dependabot.

### CODE REVIEW

File: src/infra/monitoring/grafana/Dockerfile [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+# The insertqueries script replaces the placeholders in the dashboard files with the actual KQL queries 
+WORKDIR /app/config
 RUN chmod +x insertqueries.sh && ./insertqueries.sh
 
-# Build phase is done, now build the container. Start with the office Grafana image
-# We're using 8.3.4 for now since in later versions our frontdoor origin is not accepted. 
-# This is a known issue and we'll upgrade to a later version once that's resolved. 
+# The healthmodel panel is a third party component, published on NPM. 
+# It contains only the built solution, so there is nothing to build, only install. 
+WORKDIR /app/healthmodelpanel
```
Comment (sebader) [inline src/infra/monitoring/grafana/Dockerfile:15]: so where does the healthmodel gets pulled in?

Comment (nielsams) [reply src/infra/monitoring/grafana/Dockerfile:15]: Line 16. Based on what's put in package.json.

Comment (sebader) [reply src/infra/monitoring/grafana/Dockerfile:15]: ok, maybe just add a comment when this working folder is needed

File: src/infra/monitoring/grafana/README.md [hunk 2]
```diff
@@ -30,24 +30,15 @@ Currently, authentication has been set to a username/password. Obviously this is
...
-
 ## Grafana Health Model Panel
 
-The Azure Mission-Critical health model has been implemented in Azure Log Analytics using KQL queries. This is a custom Grafana visualization panel, which can be used to visualize that health model. Its main purpose is to visualize, in an intuitive way:
-
-- The health state of each component
-- The hierarchical dependencies between components.
-
-This document describes the specifics of the custom Grafana visualization and the dependencies it has on the underlying solution. For a broader context, view the Azure Mission-Critical guidance on Azure Architecture Center.
+The Azure Mission-Critical health model has been implemented in Azure Log Analytics using KQL queries. This model is visualized using a custom component, which is managed outside of this project and thus treated as third-party. What we discuss here is the way to use it, not the way it was built.
```
Comment (sebader) [inline src/infra/monitoring/grafana/README.md:35]: can we put a link here to the Github repo of the panel?

File: src/infra/monitoring/grafana/healthmodelpanel/package.json [hunk 3]
```diff
@@ -1,37 +1,10 @@
...
-    "axios": "^0.21.4",
-    "yargs-parser": "^13.1.2"
-  }
+    "name": "alwayson-healthmodelpanel",
+    "version": "1.0.0",
+    "description": "AlwaysOn Health Model panel",
+    "author": "nielsb@microsoft.com",
+    "license": "MIT",
+    "dependencies": {
+        "@nielsams/grafana-healthmodelpanel": "^0.1.3-alpha"
```
Comment (sebader) [inline src/infra/monitoring/grafana/healthmodelpanel/package.json:8]: what will it take it get out of alpha? :)

**Review by sebader — CHANGES_REQUESTED** [review]

File: src/infra/monitoring/grafana/healthmodelpanel/package.json (hunk 3 above)
Comment (nielsams) [reply src/infra/monitoring/grafana/healthmodelpanel/package.json:8]: a first beta release :)

**Review by sebader — APPROVED** [review]
Lgtm

**Pull request merged**